modules/rosa-sts-creating-a-cluster-with-customizations-cli.adoc

```diff
@@ -280,7 +278,13 @@ Tags that are added by Red Hat are required for clusters to stay in compliance w
 {product-title} does not support adding additional tags outside of ROSA cluster-managed resources. These tags can be lost when AWS resources are managed by the ROSA cluster. In these cases, you might need custom solutions or tools to reconcile the tags and keep them intact.
 ====
 <7> Optional: Multiple availability zones are recommended for production workloads. The default is a single availability zone.
-<8> Optional: Enable this option if you are using your own AWS KMS key to encrypt the control plane, infrastructure, worker node root volumes, and PVs. Specify the ARN for the KMS key that you added to the account-wide role ARN in the preceding step.
+<8> Optional: You can create a cluster in an existing VPC, or ROSA can create a new VPC to use.
++
+[IMPORTANT]
+====
+To avoid cluster failure issues, do not install a ROSA cluster in an existing VPC created by the previous cluster's ROSA installer. If the cluster that created the VPC during the installation is deleted, the associated installer-created VPC also gets deleted, resulting in the failure of all the clusters installed in the same VPC.
```

Comment:
Hmm... I don't think the pre existing cluster can delete successfully with it's vpc in using by another one.

Reply:
@xueli181114 Are you able to verify? This is the same warning we have in other locations telling customers not to use a managed VPC for another managed cluster.